SnapChat/App/Components/AddFriends/AddFriends.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react'
+import { View, Text, TouchableHighlight, Image, ScrollView } from 'react-native'
+import { addFriendsStyles as styles } from './addFriendsStyles'
+
+const AddFriends = ({backButtonPressed}) => (
+
+    <View style={styles.container}>
+
+        {/* Header */}
+        <View style={styles.header}>
+            <TouchableHighlight
+                onPress={backButtonPressed}
+                underlayColor="#F5F5F5">
+                <Image
+                    style={styles.backArrow}
+                    source={require('../../../images/back_arrow.png')}
+                />
+            </TouchableHighlight>
+            <Text style={styles.headerTitle}>Add Friends</Text>
+            <Text style={styles.backArrow}></Text>
+        </View>
+
+        <ScrollView>
+            <Row field="Add by Username" />
+            <Row field="Add from Address Book" />
+            <Row field="Add by Snapcode" />
+            <Row field="Add Nearby" />
+            <Row field="Share username" />
+        </ScrollView>
+
+    </View>
+
+)
+
+const Row = ({field}) => {
+    return (
+        <View>
```

Comment:
Can you explain in proper english mela?? Are you trying to say that I should make another function which contains all of the Scroll View? Then only call that once: Eg. <NewView />
instead of
+        <ScrollView>
+            <Row field="Add by Username" />
+            <Row field="Add from Address Book" />
+            <Row field="Add by Snapcode" />
+            <Row field="Add Nearby" />
+            <Row field="Share username" />
+        </ScrollView>